packages/block-library/src/cover/edit.js

```diff
@@ -389,7 +415,7 @@ class CoverEdit extends Component {
 			</>
 		);
 
-		if ( ! ( url || overlayColor.color ) ) {
+		if ( ! ( url || overlayColor.color || customGradient ) ) {
```

Comment:
Looks like this combination is rechecked a lot. Might be nice it create a constant?